script/run-clang-format.py

```diff
@@ -397,8 +397,6 @@ def main():
                 continue
             if not args.quiet:
                 print_diff(outs, use_color=colored_stdout)
-            if retcode == ExitStatus.SUCCESS:
-                retcode = ExitStatus.DIFF
```

Comment:
**question:** Would this mean the exit code no longer reflect the correct state of the formatter (ie. format issues do not return a non-zero exit code)? If yes, I would propose not to modify the behaviour of the original script.

Few ideas,

1. Mark the step as `continue-on-error: true`
  ```yml
   - name: clang format check
        run: ./script/run-clang-format.py -rq .
        continue-on-error: true
   ```
2. Remove the CI step altogether. We will add it with all the format fixes in a separate PR.

I am leaning towards Option 1. WDYT?

Reply:
I think I misunderstood the word "flag", I thought the flag is the error flag for CI, just confirm is it the ` continue-on-error: true`? So like we can have this flag to make the CI pass then remove this in a follow up PR to format all code?

Reply:
> I think I misunderstood the word "flag", I thought the flag is the error flag for CI

Sorry for the confusion 🙇🏼 

> just confirm is it the `continue-on-error: true`? So like we can have this flag to make the CI pass then remove this in a follow up PR to format all code?

Yep.